index.js

```diff
@@ -66,14 +66,14 @@ function openFile (file, sonic) {
 
   if (sonic.sync) {
     try {
-      const fd = fs.openSync(file, 'a')
+      const fd = fs.openSync(file, sonic.append ? 'a' : 'w')
```

Comment:
How does this work on Windows?

> On Windows, opening an existing hidden file using the 'w' flag (either through fs.open() or fs.writeFile() or fsPromises.open()) will fail with EPERM.

Reply:
I think "hidden" is the important part here. Since this is an edge case which is not supported by Node itself, I doubt it should be a responsibility of sonic-boom to handle that, as it is consistent with overall OS behaviour.